tensorboard/plugins/distributions/tf_distribution_dashboard/tf-distribution-dashboard.html

```diff
@@ -71,61 +70,117 @@
       </div>
 
       <div class="center">
-        <tf-panes-helper
-          categories="[[_categories]]"
-          data-type="[[dataType]]"
-          data-provider="[[dataProvider]]"
-          data-not-found="[[dataNotFound]]"
-          run2tag="[[run2tag]]"
-          selected-runs="[[_selectedRuns]]"
-          repeat-for-runs
+        <template is="dom-if" if="[[_dataNotFound]]">
```

Comment:
One of these days we should refactor this error message into a single component, since we're repeating ourselves.